### ISSUE TITLE
How do I disable the system state report?

### ISSUE THREAD

**Issue by Taomyn** [EDCD/EDDI#1587]
I don't want to simply hack the script each time there's an update so I'd rather set something in my VoiceAttack initialisation that disables it at run time.

I can see in the Jumped speech responder script that these variables get set:

{set state_report to F("System state report")}
{set system_missions to F("Mission check system")}
{set system_crimes to F("Crime check system")}

How do I set "System state report" so it's "false"? I can't find this or the F() function documented or at least searching has been difficult when it's called just "F".

BTW, is there a better place to ask questions like this than reporting an issue here?

**Comment by nepomuk16321** [comment]
You can disable the "System state report" by preventing the call in the "FSD engaged" 
and "Jumped" areas.
The System state report is a function called by the following command in 
the corresponding scripts: **{F("System state report")}**
You have to find all calls in the corresponding character and switch it off with "**_**". The command will look like this: **{_ F("System state report")}**
I use notepad++. In my character these are 5 calls, 4x in "FSD engaged" and 1x in "Jumped".

**Comment by JCalvertJoshua** [comment]
I've made a command in VoiceAttack with which I can disable or enable the system state report.
To use it I've added just two lines in EDDI's "System state report" script:
At the top of that script I've added the line `{if state.statereporting!= "Off":`
and all the way down I've added a `}`

Then in VoiceAttack I've made a command called "Toggle statereporting" which looks like this:
```
Begin Text Compare : [statereporting] Equals 'On'
    Set Text [statereporting] to 'Off' (save value to profile)
Else
    Set Text [statereporting] to 'On' (save value to profile)
End Condition
Set Text [State Variable] to 'statereporting'
Execute external plugin, 'EDDI 3.5.0'

```
In the line `Execute external plugin, 'EDDI 3.5.0'` the Plugin Context should be `setstate` and the Text variable in that same line should be `State Variable`

Because I want the setting to remain the same after I quit playing and later start playing again I also edited a VA command that is activated at the startup like `((EDDI commander loading))`
In it I've put the following:
```
Set Text [statereporting] to [Saved Value] (save value to profile)
Set Text [State Variable] to 'statereporting'
Write [Blue] 'Statereporting : {TXT:statereporting}' to log
Execute external plugin, 'EDDI 3.5.0' and wait for return
```
The settings in the last line are the same as I mentioned earlier.

So now when I get tired of hearing the state report at every jump I just say "Toggle statereporting" and it gets turned off until I repeat that command during the same playing session or a later one.

And when a new version of EDDI is released I only have to check if the "System state report" script has been changed and, if it has, modify the new version again if necessary.

So far, and I'm sure there are more elegant solutions, this works for me.
I hope my English makes sense :-)
